### DISCUSSION TITLE
How to find the caller of method while tracing ( I used filtermethod and I want to find the caller of filtered method)

### DISCUSSION THREAD

**Discussion by Mandofskii** [vfsfitvnm/frida-il2cpp-bridge#211] [Q&A]
.

**Comment by bluewave41** [comment]
As far as I know there aren't any simple code methods for this.

https://github.com/vfsfitvnm/frida-il2cpp-bridge/issues/43#issuecomment-934539856

The Ghidra option is probably the easiest as it doesn't involve guessing.
